pdp10-opc.c

```diff
@@ -322,497 +323,497 @@ const struct pdp10_instruction pdp10_instruction[] =
+  { "hllzs",	0513,	PDP10_BASIC,	PDP10_ALL, 0, 0 },
+  { "hrlz",	0514,	PDP10_BASIC,	PDP10_ALL, 0, 0 },
+  { "hrlzi",	0515,	PDP10_BASIC,	PDP10_ALL, 0, HINT_IMMEDIATE },
+  { "hrlzm",	0516,	PDP10_BASIC,	PDP10_ALL, 0, 0 },
+  { "hrlzs",	0517,	PDP10_BASIC,	PDP10_ALL, 0, 0 },
+  { "hllo",	0520,	PDP10_BASIC,	PDP10_ALL, 0, 0 },
+  { "hlloi",	0521,	PDP10_BASIC,	PDP10_ALL, 0, HINT_IMMEDIATE },
+  { "hllom",	0522,	PDP10_BASIC,	PDP10_ALL, 0, 0 },
+  { "hllos",	0523,	PDP10_BASIC,	PDP10_ALL, 0, 0 },
+  { "hrlo",	0524,	PDP10_BASIC,	PDP10_ALL, 0, 0 },
+  { "hrloi",	0525,	PDP10_BASIC,	PDP10_ALL, 0, HINT_IMMEDIATE },
+  { "hrlom",	0526,	PDP10_BASIC,	PDP10_ALL, 0, 0 },
+  { "hrlos",	0527,	PDP10_BASIC,	PDP10_ALL, 0, 0 },
+  { "hlle",	0530,	PDP10_BASIC,	PDP10_ALL, 0, 0 },
+  { "hllei",	0531,	PDP10_BASIC,	PDP10_ALL, 0, HINT_IMMEDIATE },
+  { "hllem",	0532,	PDP10_BASIC,	PDP10_ALL, 0, 0 },
+  { "hlles",	0533,	PDP10_BASIC,	PDP10_ALL, 0, 0 },
+  { "hrle",	0534,	PDP10_BASIC,	PDP10_ALL, 0, 0 },
+  { "hrlei",	0535,	PDP10_BASIC,	PDP10_ALL, 0, HINT_IMMEDIATE },
+  { "hrlem",	0536,	PDP10_BASIC,	PDP10_ALL, 0, 0 },
+  { "hrles",	0537,	PDP10_BASIC,	PDP10_ALL, 0, 0 },
+  { "hrr",	0540,	PDP10_BASIC,	PDP10_ALL, 0, 0 },
+  { "hrri",	0541,	PDP10_BASIC,	PDP10_ALL, 0, HINT_IMMEDIATE },
+  { "hrrm",	0542,	PDP10_BASIC,	PDP10_ALL, 0, 0 },
+  { "hrrs",	0543,	PDP10_BASIC,	PDP10_ALL, 0, 0 },
+  { "hlr",	0544,	PDP10_BASIC,	PDP10_ALL, 0, 0 },
+  { "hlri",	0545,	PDP10_BASIC,	PDP10_ALL, 0, HINT_IMMEDIATE },
+  { "hlrm",	0546,	PDP10_BASIC,	PDP10_ALL, 0, 0 },
+  { "hlrs",	0547,	PDP10_BASIC,	PDP10_ALL, 0, 0 },
+  { "hrrz",	0550,	PDP10_BASIC,	PDP10_ALL, 0, 0 },
+  { "hrrzi",	0551,	PDP10_BASIC,	PDP10_ALL, 0, HINT_IMMEDIATE },
+  { "hrrzm",	0552,	PDP10_BASIC,	PDP10_ALL, 0, 0 },
+  { "hrrzs",	0553,	PDP10_BASIC,	PDP10_ALL, 0, 0 },
+  { "hlrz",	0554,	PDP10_BASIC,	PDP10_ALL, 0, 0 },
+  { "hlrzi",	0555,	PDP10_BASIC,	PDP10_ALL, 0, HINT_IMMEDIATE },
+  { "hlrzm",	0556,	PDP10_BASIC,	PDP10_ALL, 0, 0 },
+  { "hlrzs",	0557,	PDP10_BASIC,	PDP10_ALL, 0, 0 },
+  { "hrro",	0560,	PDP10_BASIC,	PDP10_ALL, 0, 0 },
+  { "hrroi",	0561,	PDP10_BASIC,	PDP10_ALL, 0, HINT_IMMEDIATE },
+  { "hrrom",	0562,	PDP10_BASIC,	PDP10_ALL, 0, 0 },
+  { "hrros",	0563,	PDP10_BASIC,	PDP10_ALL, 0, 0 },
+  { "hlro",	0564,	PDP10_BASIC,	PDP10_ALL, 0, 0 },
+  { "hlroi",	0565,	PDP10_BASIC,	PDP10_ALL, 0, HINT_IMMEDIATE },
+  { "hlrom",	0566,	PDP10_BASIC,	PDP10_ALL, 0, 0 },
+  { "hlros",	0567,	PDP10_BASIC,	PDP10_ALL, 0, 0 },
+  { "hrre",	0570,	PDP10_BASIC,	PDP10_ALL, 0, 0 },
+  { "hrrei",	0571,	PDP10_BASIC,	PDP10_ALL, 0, HINT_IMMEDIATE },
+  { "hrrem",	0572,	PDP10_BASIC,	PDP10_ALL, 0, 0 },
+  { "hrres",	0573,	PDP10_BASIC,	PDP10_ALL, 0, 0 },
+  { "hlre",	0574,	PDP10_BASIC,	PDP10_ALL, 0, 0 },
+  { "hlrei",	0575,	PDP10_BASIC,	PDP10_ALL, 0, HINT_IMMEDIATE },
+  { "hlrem",	0576,	PDP10_BASIC,	PDP10_ALL, 0, 0 },
+  { "hlres",	0577,	PDP10_BASIC,	PDP10_ALL, 0, 0 },
+  { "trn",	0600,	PDP10_BASIC,	PDP10_ALL, 0, 0 },
```

Comment:
Should the Txxx instructions have HINT_IMMEDIATE too? (I guess there could be a HINT_BITMASK in the future...)